source/partials/drupal/dependencies-compatible.md

```diff
@@ -0,0 +1,29 @@
+---
+contenttype: [partial]
+categories: [--]
+newcms: [drupal, drupal8, drupal9]
+product: [--]
+integration: [--]
+tags: [--]
+reviewed: "2022-11-04"
+---
+
+You must confirm that your site is compatible with the latest version of Drupal before you change the Drupal core version.
+ 
+Review the steps below to confirm compatibility for your site's components.
+
+1. Optional. [Convert your site to Composer](/guides/composer-convert) if you plan to use the [Upgrade Status module](https://www.drupal.org/project/upgrade_status/) to assist in your upgrade.
```

Comment:
I don't think this is optional any longer, as Pantheon requires Composer for Drupal 9+. I'd just remove "Optional"

Reply:
done